rpc/core/src/lib.rs

```diff
@@ -9,6 +22,7 @@ pub mod notify;
 pub mod wasm;
 
 pub mod prelude {
+    //! Glob imports of the most commonly used types and traits in this crate.
```

Comment:
`Glob` appears here again

Reply:
Glob is sometimes used as a reference to wildcards.  

Replaced wording with "Re-export".